compiler/rustc_attr_data_structures/src/attributes.rs

```diff
@@ -153,7 +154,9 @@ impl Deprecation {
 /// the place where `must_use` is checked) little to no extra parsing or validating needs to
```

Comment:
Tangentially related: Is `#[must_use]` represented as `hir::Attribute::Unparsed`? It's not clear where it's defined in this enum.

Reply:
Haha, no it's just a good example but I didn't get to it yet. But, that's an easy one I might just do it tomorrow or something like that :)

Reply:
I believe the reference also used must use as an example so that matches somewhat nicely